### PR TITLE
Switch to consistent second person

### PR DESCRIPTION
**Pull request by Benjamin-Lee**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**
As part of #173, @michaelmhoffman pointed out that we were switching between second person and awkward third person. I have switched to using second person everywhere to improve the flow and ensure a consistent voice.

### CODE REVIEW

File: content/08.hyperparameters.md [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
+In conventional implementations, the relative weighting of each component is a function of the number of input features (more increase the importance of reconstruction loss) and the number of features in the latent space (more increase the importance of the distribution loss).
 **{SR: I am not sure this is correct, why would a larger number of, e.g., pixels make the reconstruction loss more important? I suppose this is true if we just sum over the pixel-wise differences, but if we average, e.g., using MSE, I am not convinced this is true. Please comment.}**
-Users who apply a VAE architecture to a new dataset with more input features, even without changing any hyperparameters, alter the relative weights of the components of the loss function.
+Users who apply a VAE architecture to a new dataset with more input features, even without changing any hyperparameters, alter the relative weights of the components of the loss function. **{BDL: I'm not entirely sure this sentence makes sense, nor how to make this second person.}**
```
Comment (agitter) [inline content/08.hyperparameters.md]: Given your comment and the comment on the sentence above, I'd say this paragraph needs some reworking.  That can be done in another PR.

File: content/09.overfitting.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Tip 7: Address deep neural networks' increased tendency to overfit the dataset {#overfitting}
 
-Overfitting is one of the most significant dangers faced by a deep learning practitioner.
+Overfitting is one of the most significant dangers you'll face in deep learning (and traditional deep learning).
```
Comment (agitter) [inline content/09.overfitting.md]: `traditional machine learning`?

Comment (Benjamin-Lee) [reply content/09.overfitting.md]: Good eye!

**Comment by rasbt** [comment]
Before approving/merging, is 2nd person a common convention for the journal we are submitting to?

**Comment by Benjamin-Lee** [comment]
[This paper](doi.org/10.1371/journal.pcbi.1006915), along with five of the other papers in the "*n* quick tips" series I looked at all use "you" in their bodies, so I think we should be fine.

**Comment by rasbt** [comment]
> [This paper](doi.org/10.1371/journal.pcbi.1006915), along with five of the other papers in the "_n_ quick tips" series I looked at, all use "you" in their bodies, so I think we should be fine.

Sounds good then, thanks for double-checking.

**Pull request merged**